2.0.0-beta (#13)
sethgoldin/s3-xxHash-transfer: README.md

   1. Depending on your OS:
      1. On macOS or Linux, run `$ sh send.sh`. `sudo` privilege is _not_ required.
      1. On Windows, run `PS> send.ps1`.

>    1. The "send" script will prompt for the source directory, "seal" the contents of the directory with 64-bit xxHash checksums, prompt for the S3 URL, and then ingest the entire directory into the bucket.

Author: ```suggestion
   1. The "send" script will prompt for the source directory, "seal" the contents of the directory with 64-bit xxHash checksums, prompt for an S3 URL, and then ingest the entire directory into the bucket.
```

1. On the "destination" computer:
   1. Depending on your OS:
      1. On macOS or Linux, run `$ sh receive.sh`. `sudo` privilege is _not_ required.
      1. On Windows, run `PS> receive.ps1`.

>    1. The "receive" script will prompt for the S3 URL that had been specified by the "send" script, prompt for the local directory path into where the data will be downloaded, and then will automatically download the specified data from the S3 bucket and verify its 64-bit xxHash checksums.

Author: ```suggestion
   1. The "receive" script will prompt for the S3 URL specified earlier by the "send" script, prompt for the local directory path into where the data will be downloaded, and then will automatically download the specified data from the S3 bucket and verify their 64-bit xxHash checksums.
```

- The [`.pkg` installer from Pomfort](http://download.pomfort.com/mhl-tool.zip) will install a precompiled binary for macOS into `/usr/local/bin`, which is included by default in macOS's `$PATH`.
- On Windows, download and extract [the precompiled binary from Pomfort](http://download.pomfort.com/mhl-tool.zip), and then copy or move `mhl.exe` into `C:\Windows\System32\`, which is included by default in the Windows `Path` system environment variables.
- The [AWS CLI](https://aws.amazon.com/cli/) should be installed and configured on both endpoints, with:
  - The sending IAM user having permission to write into the S3 bucket
  - The receiving IAM user having permission to read from the S3 bucket
  - Both endpoints connected to the same [region](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions)
  - The command output format set to [text](https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-output.html#text-output)

## Tested platforms

> Release 2.0.0-alpha.1 has been tested on Linux, macOS, and Windows, specifically on:

Author: ```suggestion
Release 2.0.0-beta has been tested on Linux, macOS, and Windows, specifically on:
```
